app/s2i/pom.xml

```diff
@@ -257,5 +261,18 @@
         <mavenMirrorOf>external:*</mavenMirrorOf>
       </properties>
     </profile>
+    <!-- Build by PNC -->
+    <profile>
+      <id>pnc</id>
```

Comment:
```suggestion
      <id>with-proxy</id>
```

Perhaps?

Reply:
Yeah, I see that the same exists on `ui-react` module: https://github.com/syndesisio/syndesis/blob/master/app/ui-react/pom.xml#L149 - I think such a profile could be ported to the parent pom. Any inconvenience if I go on that direction?

Reply:
The previous idea is not feasible as it would imply porting also variables in the parent pom, sticking to the `with-proxy` name instead.